README.md

```diff
@@ -1,6 +1,43 @@
 # asq-java-q-backend
+
+### Info
+
 Backend for sandboxed Java execution for the asq-java-q question type
 
 To run the service, you need docker running on the machine.
 It uses /var/run/docker.socket to connect to it, so on windows it may not be working.
 
+Before running the server the first time do:
```

Comment:
Oh, I see. You can add an additional `install` step to the `npm install` lifecycle, so that you run the `docker pull openjdk` at install time of your package.